extensions/paredit-mode/paredit-mode.lisp

```diff
@@ -309,16 +312,19 @@ link : http://www.daregada.sakuraweb.com/paredit_tutorial_ja.html
           ((syntax-escape-point-p p 0)
            (insert-character p c))
           ((char= c (character-at p))
-           (if (syntax-escape-point-p p 0)
-               (insert-character p c)
-               (forward-char)))
+           (when *remove-whitespace*
+             (with-point ((from p))
+               (skip-whitespace-backward from)
+               (delete-between-points from p)))
+           (forward-char))
           ((ignore-errors (or (scan-lists p 1 1)) t)
            (with-point ((new-p p))
              (character-offset new-p -1)
              (move-point (current-point) new-p)
              (with-point ((p new-p))
                (skip-whitespace-backward p)
-               (delete-between-points p new-p))))
+               (delete-between-points p new-p)
+               (forward-char))))
```

Comment:
In this case, it is better to use character-offset than forward-char.

```lisp
(character-offset p 1)
```

Reply:
It was already used above in the same function. I guess I'll change both to `character-offset`.